### DISCUSSION TITLE
Partial Hazard interpretation in CoxTimeVaryingFitter

### DISCUSSION THREAD

**Discussion by EleniNeti** [CamDavidsonPilon/lifelines#1640] [Q&A]
Hi everyone,
I have brought my dataset in the long format as suggested by the documentation. Problem arises when I do the predict_partial_hazard method, since I find it hard to interpret the output precisely. Does each prediction refer to _each_ patient? I'm confused since there can be multiple instances referring to the same patient ID. Can someone that used it before give some help regarding the interpretation?

**Comment by AhmetZamanis** [comment]
Hi,

The `predict_partial_hazard` method will give you the estimated partial hazard of each observation, based on their covariate values. If a patient has multiple observations / measurements in your dataset, with different covariate values, the estimated partial hazard be different for each observation. 

How you interpret that depends on what you need to analyze. In a project, I plotted the estimated partial hazard values against covariate values to try & understand how the covariates affect the partial hazard estimates.
